presidio-image-anonymizer/tests/test_image_analyzer_engine.py

```diff
@@ -1,11 +1,13 @@
 import pytest
-from presidio_image_anonymizer.image_analyzer_engine import ImageAnalyzerEngine
-from presidio_image_anonymizer.image_recognizer_result import ImageRecognizerResult
+
 from presidio_analyzer import RecognizerResult
+from presidio_image_anonymizer.entities.image_recognizer_result import \
+    ImageRecognizerResult
+from presidio_image_anonymizer.image_analyzer_engine import ImageAnalyzerEngine
 
 
 def test_given_valid_ocr_and_entities_then_map_analyzer_returns_correct_len_and_output(
-    get_ocr_analyzer_results, get_image_recognizerresult
+        get_ocr_analyzer_results, get_image_recognizerresult
 ):
     ocr_result, text, recogniser_result = get_ocr_analyzer_results

```

Comment:
```suggestion
    ocr_result, text, recognizer_result = get_ocr_analyzer_results

```

Reply:
Just a nit - there are a few more of these throughout this file

Reply:
I will do it in another commit. :)